module/zfs/arc.c

```diff
@@ -468,6 +468,24 @@ static int zfs_arc_prune_task_threads = 1;
 /* Used by spa_export/spa_destroy to flush the arc asynchronously */
 static taskq_t *arc_flush_taskq;
 
+/*
+ * Controls the number of ARC eviction threads.
+ * Possible values:
+ * 0  (disabled) parallel eviction threads are not used.
+ * 1  (auto) compute the number of threads using a logarithmic formula.
```

Comment:
This logic seems backwards. Other tunables compute a default value when set to zero, otherwise use the supplied value.

Reply:
Ok, if that is less confusing, I can swap the meanings of 0/1.